src/image/utils/schema/triggers.py

```diff
@@ -23,6 +27,14 @@ class ImageUploadReleaseSchema(pydantic.BaseModel):
     class Config:
         extra = pydantic.Extra.forbid
 
+    @pydantic.validator("end_of_life")
+    @classmethod
+    def ensure_still_supported(cls, v: datetime) -> datetime:
+        """ensure that the end of life isn't reached."""
+        if v < datetime.now(timezone.utc):
+            raise ImageReachedEol("This image revision has reached its end of life")
```

Comment:
```suggestion
            raise ImageReachedEol("This track has reached its end of life")
```

Reply:
what about https://github.com/canonical/oci-factory/pull/119/files#diff-120dfed5520cc4aaf924c8389d168eed6e1dc2e75affe683763c3f1de50e6d86R54?

Reply:
sure thing. I'll add it there too.